src/platformio.ini

```diff
@@ -1,467 +1,21 @@
-; PlatformIO Project Configuration File
-;
-;   Build options: build flags, source filter
-;   Upload options: custom upload port, speed and extra flags
-;   Library options: dependencies, extra library storages
-;   Advanced options: extra scripting
-;
-; Please visit documentation for the other options and examples
-; https://docs.platformio.org/page/projectconf.html
+; ExpressLRS PlatformIO Project Configuration File
+
+[platformio]
+data_dir = .
+extra_configs =
+	targets/common.ini
+	targets/frsky/frsky.ini
+	targets/happymodel/happymodel.ini
+	targets/imrc/imrc.ini
+	targets/namimnorc/namimnorc.ini
+	targets/neutronrc/neutronrc.ini
+	targets/siyi/siyi.ini
+	targets/diy/diy_900.ini
+	targets/diy/diy_2400.ini
```

Comment:
Change those lines in `targets/*/*.ini`

Reply:
Hmmm, need to think this...
New manufacturers are not added very often. And that way we can ignore some specific ones if needed. 
Good point anyway!

Reply:
Yes, I’m thinking about the future. This is a very valid project so it can easily become like betaflight with plenty of manufacturer with their own hardwares 😉

Reply:
this is awesome project! 
...Then we need to start thinking of unified targets 🙃
I was thinking more like a case where we want to stop showing long list of e.g. DIY target in the "main" or similar... That is not possible if we use wildcards.. ofc there is other ways to do the same :)
@AlessandroAU what do you think?

Those target specific ini files can be used to build just those envs defined in target ini. This way users are able to limit shown list... Like you said, there could be pretty long list in the future 😄